roles/generate-docs/README.md

```diff
@@ -0,0 +1,109 @@
+gen-docs
+========
+
+The purpose of this role is to auto-generate documentation for labs residencies.
+
+Requirements
+------------
+
+For testing purposes, you may want to consider using `grip`. This will allow
+you to use fully rendered markdown files.
+
+https://pypi.org/project/grip/
+
+Role Variables
+--------------
```

Comment:
I think the only role variable you may is the `template` name/path. Other than that, the input variables/facts will depend on the template content, so maybe worth nothing that here.